### PR TITLE
PR for suggesting frequency change of LAMMPS and image dump command commented for bigred3

### PR DESCRIPTION
**Pull request by kadupitiya**
@jadhao: PR for suggesting frequency change of LAMMPS and image dump command commented for bigred3.

### CODE REVIEW

**Comment by kadupitiya** [comment]
@jadhao: PR sent. We can go over this and approve it.

**Review by jadhao — COMMENTED** [review]
@kadupitiya this looks good

File: bin/infiles/in.lammps.chargedsurface.template [hunk 1]
```diff
@@ -15,6 +15,18 @@
 # epsilon_water = 80;
 # room_temperature = 298 in Kelvin;
 # cutoff for LJ interactions = 2 ^ (1/6) = 1.122462048 (in reduced lj unit);
+
+# ------------------------------ USER DEFINED PARAMETERS -----------------------------------
+variable detlatT equal USERINPUT_TIMESTEP
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:20]: @kadupitiya detlatT to deltaT

File: bin/infiles/in.lammps.chargedsurface.template [hunk 2]
```diff
@@ -59,33 +71,45 @@ fix wallloo     negative_ions wall/lj126 zlo EDGE 1.0  NEGATIVE_RADIUS  HALF_NEG
...
 fix               equilibrate salt_ions nvt temp 1.0 1.0 0.1                      # This updates the position and velocity for atoms in the group each timestep in the canonical (nvt) ensemble. # T_start, T_stop, T_damp=100*timestep: The desired temperature at each timestep is a ramped value during the run from Tstart to Tstop. The Tdamp parameter is specified in time units and determines how rapidly the temperature is relaxed;
-timestep        USERINPUT_TIMESTEP                                                # The timestep command specifies this dimensionless multiplier as the magnitude of the timestep relative to (tau);
-thermo	        USERINPUT_THERMO_DUMP_FRQ                                          # How often does the data get dumped
-run             USERINPUT_STEPS_BEFORE_EQ                                         # Run time for equilibration;
+timestep        ${detlatT}                                                        # The timestep command specifies this dimensionless multiplier as the magnitude of the timestep relative to (tau);
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:81]: @kadupitiya detlatT to deltaT

File: bin/infiles/in.lammps.chargedsurface.template [hunk 3]
```diff
@@ -15,6 +15,18 @@
...
 # cutoff for LJ interactions = 2 ^ (1/6) = 1.122462048 (in reduced lj unit);
+
+# ------------------------------ USER DEFINED PARAMETERS -----------------------------------
+variable detlatT equal USERINPUT_TIMESTEP
+variable thermoDumpFrq equal USERINPUT_THERMO_DUMP_FRQ
+variable stepsBeforeEq equal USERINPUT_STEPS_BEFORE_EQ
+variable stepsAfterEq equal USERINPUT_STEPS_AFTER_EQ
+variable movieFrq equal USERINPUT_MOVIE_FRQ
+variable totalSteps equal ${stepsBeforeEq}+${stepsAfterEq}
+variable densityDataFrq equal 100
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:26]: @kadupitiya could set 100 to freq param from inhouse

File: bin/infiles/in.lammps.chargedsurface.template [hunk 4]
```diff
@@ -59,33 +71,45 @@ fix wallloo     negative_ions wall/lj126 zlo EDGE 1.0  NEGATIVE_RADIUS  HALF_NEG
...
+thermo	        ${thermoDumpFrq}                                                  # How often does the data get dumped
+run             ${stepsBeforeEq}                                         		  # Run time for equilibration;
 write_restart   outfiles/equilibrated.electrolyte.system.*                        # Restart command to create a sample of fluid with which you can write a fix
 unfix           equilibrate                                                       # Similar to a bracket close
-dump	          mymovie  salt_ions    custom	USERINPUT_MOVIE_FRQ outfiles/electrolyte_movie.xyz id  type  q  x   y   z
+dump	          mymovie  salt_ions    custom	${movieFrq} outfiles/electrolyte_movie.xyz id  type  q  x   y   z
 
 ## Image Dump ## ## removing images to work in bigred3 -- does not affect nanohub as it does not have charged surfae ##
 #dump	myimages all image 10000 outfiles/image.*.jpg type type size 1024 1024 &
 #               	axes no 0.8 0.02 view 60 -30
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:90]: @kadupitiya make this one line remove the & from above and append axes... to the above line.

File: bin/infiles/in.lammps.chargedsurface.template [hunk 5]
```diff
@@ -59,33 +71,45 @@ fix wallloo     negative_ions wall/lj126 zlo EDGE 1.0  NEGATIVE_RADIUS  HALF_NEG
...
 #dump_modify myimages boxcolor black backcolor white
 
+# --------------------------------- Desnity profile dump --------------------------------------
+
+#compute ID group-ID chunk/atom dim origin delta #units reduced
+compute pdpbin positive_ions chunk/atom bin/1d z center ${binWidth}
+compute ndpbin negative_ions chunk/atom bin/1d z center ${binWidth}
+
+#For example, if Nevery=2, Nrepeat=6, and Nfreq=100, then values on timesteps 90,92,94,96,98,100 will be used to compute the final average on timestep 100. 
+fix postivedensity positive_ions ave/chunk ${densityDataFrq} ${densityDataSamples} ${totalSteps} pdpbin density/number ave one file data/p_density_profile.lmp.dat
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:101]: @kadupitiya postivedensity to positivedensity

File: bin/infiles/in.lammps.chargedsurface.template [hunk 6]
```diff
@@ -59,33 +71,45 @@ fix wallloo     negative_ions wall/lj126 zlo EDGE 1.0  NEGATIVE_RADIUS  HALF_NEG
...
-timestep        USERINPUT_TIMESTEP                                                # The timestep command specifies this dimensionless multiplier as the magnitude of the timestep relative to (tau).
-thermo          USERINPUT_THERMO_DUMP_FRQ                                          # How often does the data get dumped
-run             USERINPUT_STEPS_AFTER_EQ                                          # Run time after equilibration;
+timestep        ${detlatT}                                                        # The timestep command specifies this dimensionless multiplier as the magnitude of the timestep relative to (tau).
+thermo          ${thermoDumpFrq}                                                  # How often does the data get dumped
+run             ${stepsAfterEq}                                                   # Run time after equilibration;
 write_restart   outfiles/large.sample.electrolyte.*                               # Restart command to create a sample of fluid with which you can write a fix
 unfix           heat                                                              # Similar to a bracket close
+unfix           postivedensity
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:111]: @kadupitiya postivedensity to positivedensity

File: bin/infiles/in.lammps.chargedsurface.template [hunk 7]
```diff
@@ -59,33 +71,45 @@ fix wallloo     negative_ions wall/lj126 zlo EDGE 1.0  NEGATIVE_RADIUS  HALF_NEG
...
+unfix           negativedensity
```
Comment (jadhao) [inline bin/infiles/in.lammps.chargedsurface.template:113]: @kadupitiya do the uncompute pdpbin and uncompute ndpbin

**Pull request merged**